ReactWindows/ReactNative.Shared/DevSupport/DisabledDevSupportManager.cs

```diff
@@ -65,10 +70,17 @@ public string JavaScriptBundleUrlForRemoteDebugging
 
         public async void HandleException(Exception exception)
         {
+#if WINDOWS_UWP
             await CoreApplication.MainView.CoreWindow.Dispatcher.RunAsync(CoreDispatcherPriority.High, () =>
             {
                 ExceptionDispatchInfo.Capture(exception).Throw();
             }).AsTask().ConfigureAwait(false);
+#else
+            await Dispatcher.CurrentDispatcher.InvokeAsync(() =>
+            {
+                ExceptionDispatchInfo.Capture(exception).Throw();
+            }, DispatcherPriority.Send).Task.ConfigureAwait(false);
```

Comment:
It's just exception behavior, I actually don't think we need to use the CoreDispatcherPriority there...

---
In reply to: [82702918](https://github.com/ReactWindows/react-native-windows/pull/760#discussion_r82702918) [](ancestors = 82702918)